dev/com.ibm.ws.jaxrs.2.0.client_fat/build.gradle

```diff
@@ -0,0 +1,102 @@
+/*******************************************************************************
+ * Copyright (c) 2018 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+Properties props = new Properties()
+rootProject.file('generated.properties').withInputStream { props.load(it) }
+
+repositories {
+  /*if ((props.getProperty("artifactory.download.server") != null) && !props.getProperty('artifactory.force.external.repo')) {
```

Comment:
can we delete the comments?  or is there some value in having them?

Reply:
that is going to be changed in the future when authorization for the mirror is fixed